tox.ini

```diff
@@ -158,6 +158,9 @@ deps =
   sphinx-autodoc-typehints~=1.10.2
   opentracing~=2.2.0
   Deprecated>=1.2.6
+  thrift>=0.10.0
```

Comment:
what's the choice to have some be greater than ranges while some are compatible releases?

Reply:
I just went with the same requirement as the ext package https://github.com/open-telemetry/opentelemetry-python/blob/9fa92f3b30848f0c00df1cd7afbc714fb97acc78/ext/opentelemetry-ext-jaeger/setup.cfg#L42

Reply:
ah I see. Wouldn't we get these as transitive dependencies as jeager is installed anyway?

I would argue that, for the sake of us better understanding what changed, it would be preferable to hard or narrow-pin our versions. There's been several examples of builds breaking due to external version changes.